cement/ext/ext_generate.py

```
@@ -44,14 +121,20 @@ def _generate(self, source: str, dest: str) -> None:
         g_config = yaml_load(f)
         f.close()
 
-        vars = g_config.get('variables', {})
+        vars = g_config.get('variables', [])
```

Ruff (0.15.12)

[error] Variable `vars` is shadowing a Python builtin

(A001)